### DISCUSSION TITLE
Payment issue

### DISCUSSION THREAD

**Discussion by RajeshDaroji** [community/community#61821] [New to GitHub]
payment issue not able to check current bill for payment

**Comment by RajeshDaroji** [comment]
pay now option is not showing

**Reply by DawanShamil** [reply]
I have the same issue

**Comment by queenofcorgis** [comment]
Hi @RajeshDaroji ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
